packages/editor-ui/src/views/NodeView.vue

```diff
@@ -395,6 +404,32 @@ export default mixins(
 			document.removeEventListener('keyup', this.keyUp);
 		},
 		methods: {
+			onCreateMenuHoverIn() {
+				const vm = this;
```

Comment:
is `vm` const needed? I think if we just used `this` that would work

Reply:
Good catch. I think it's a leftover of some older code I had there.